my-app/src/shared/components/Input.js

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
 import TextField from '@mui/material/TextField';
+import SearchIcon from '@mui/icons-material/Search';
+import { Box } from '@mui/material';
+import { InputAdornment } from '@mui/material';
 
 export class Input extends React.Component {
   render() {
     return (
-      <TextField
-        id="outlined-search"
-        type="search"
-        onChange={this.props.handleTextChange}
-      />
+      <Box>
+        <TextField
+          id="outlined-search"
+          type="search"
+          sx={{ width: '50%', mb: 2 }}
```

Comment:
please move all of the styles to a separate file `Input.styles.js`, also please create a folder named `Input` in wich you will have three files `Input.js` `Input.styles.js` `index.js which will only contain export of the component`.

Same for all other components in this pr